docs/topics/braintree.rst

```diff
@@ -227,6 +227,17 @@ solitude.
 
     :status 201: payment method created.
 
+Cancel a subscription. This will cancel the subscription in Braintree and is
+not reversible.
+
+.. http:post:: /braintree/subscription/cancel/
+
+    :<json string paymethod: the resource_uri of the subscription in solitude.
```

Comment:
a URL like that does require you making up a special URL though, i'm going to make the assumption that the URL to all the endpoints are stored in payments-service and that code that just says pass these two existing variables is easier to actually implement.